draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1873,17 +1873,88 @@ still provide adequate output for incomplete logs.
 
 # Security and privacy considerations {#privacy}
 
-TODO : discuss privacy and security considerations (e.g., what NOT to log, what to
-strip out of a log before sharing, ...)
-
-TODO: strip out/don't log IPs, ports, specific CIDs, raw user data, exact times,
-HTTP HEADERS (or at least :path), SNI values
-
-TODO: see if there is merit in encrypting the logs and having the server choose an
-encryption key (e.g., sent in transport parameters)
-
-Good initial reference: [Christian Huitema's
-blogpost](https://huitema.wordpress.com/2020/07/21/scrubbing-quic-logs-for-privacy/)
+Modern protocols such as QUIC ({{!RFC9000}}) take pains to encrypt a large part
+of their wire image, leading to generally strong privacy and security
+characteristics, especially considering monster-in-the-middle threat models. A
+generalized, plaintext logging scheme such as qlog by its nature undoes some of
+these characteristics to make (parts of) the protocols observable again. This is
+needed for a variety of use cases such as (live) operational insight,
+implementation debuggability or research, but carries inherent security and
+privacy risks, especially if qlog data can be correlated with other data
+sources. As such, qlog operators and implementers should be aware of these risks
+inherent in logging, storing, using, and in some cases purposefully disclosing
+potentially sensitive information as part of qlog data.
+
+A non-exhaustive list of potentially sensitive fields in qlog data include:
```

Comment:
This is a good starter but it seems a little too informal and at the same too specific in citing QUIC. Also modern will soon be old :)

Here's an alternative that builds on what you've put down already

> Protocols such as TLS [cite] and QUIC [cite] provide secure protection, to varying degrees, for the wire image [cite  RFC 8546]. There is inevitably tension between security and observability, when logging can reveal aspects of the wire image, that would ordinarily be protected. This tension equally applies to any privacy considerations that build on security properties, especially if data can be correlated across data sources.

> qlog operators and implementers should be mindful of the security and privacy risks inherent in handling qlog data. This includes but is not limited to logging, storing, or using the data. Data might be considered as non-sensitive, potentially-sensitive, or sensitive; applying the considerations in this section may produce different risks depending on the nature of the data itself, or its handling. However, in many cases the largest risk factors arise from data that can be considered as potenially-sensitive or sensitive. The following is a non-exhaustive list of such fields that can be carried in qlog data: